### PR TITLE
optode svu version

### PR DESCRIPTION
**Pull request by tomhull**
small notes and some updates to unicode.

### CODE REVIEW

**Review by soerenthomsen — APPROVED** [review]

**Comment by soerenthomsen** [comment]
@s-pearce do you have capacity to review this PR?

File: sections/oxygen_sensors_integration.md [hunk 1]
```diff
@@ -30,27 +30,37 @@ Otherwise foils should typically not be replaced unless mechanically damaged (li
...
 The way optode foils are initially calibrated by Aanderaa, and how the measured values are processed by the optode varies between different optode versions.
 The optode illuminates the sensing foil with both a red and blue LED. 
-Since the red light does not produce fluorescence in the foil the phase measurements are obtained from the difference between the blue (P1) and the red (P2) excitation.
+Since the red light does not produce fluorescence in the foil the phase measurements are obtained from the difference between the blue (`P1`) and the red (`P2`) excitation.
 
-P_T = A(T) + (P1 - P2) · B(T)
+    TCphase = A(T) + (P1 - P2) · B(T)
 
-Where P_T is the temperature compensated phase (known as ‘TCphase‘). 
+Where `TCphase` is the temperature compensated phase and `T` is the measured optode temperature temperature.
```
[the remark this answers is not in the document]
Comment (soerenthomsen) [reply sections/oxygen_sensors_integration.md:37]: @tomhull you should be able to just click "Commit suggestion" or?

**Review by soerenthomsen — COMMENTED** [review]
Don't feel knowledgeable to review this fully. 

This PR relates to #143

**Comment by soerenthomsen** [comment]
@gkrahmann are you maybe able to review this PR?

File: sections/oxygen_sensors_integration.md [hunk 2]
```diff
@@ -62,21 +72,21 @@ After this it should be less than 0.5 % per year, unless the foil is mechanicall
...
-They also use further foil design (~1 s response) which they call fast. 
-The RBR sensor has a smaller form factor than the Aandera optodes, but is overall more similar to a 4831 with the temperature sensor very closely located to the sensing foil. 
+They also use further foil design (~1 s response) which they call “fast” . 
+The RBR sensor has a smaller form factor than the Aandera optodes, but is overall similar to a 4831 with the temperature sensor very closely located to the sensing foil. 
 This sensor has recently been implemented in gliders, and little is known about their performance.
 
 ### JFE Advantech RINKO
 AROD-FT sensor (RINKO JFE) is used for the SeaExplorer gliders (Alseamar) and for some Argo floats (small size and low power consumption) (see {numref}`ARODFT`). 
-This sensor is based on the optical (phosphorescence) principle which is now widely known as a remarkably fast response oxygen sensor (below 1s) with a high accuracy of 2 $\mu$mol/kg. 
+This sensor is based on the optical (phosphorescence) principle which a remarkably fast response oxygen sensor (below 1 s) with a good accuracy (±2 μmol kg<sup>-1</sup>).
```
Comment (gkrahmann) [inline sections/oxygen_sensors_integration.md:87]: should this be
'principle creating a remarkably'
?

File: sections/oxygen_sensors_integration.md [hunk 3]
```diff
@@ -62,21 +72,21 @@ After this it should be less than 0.5 % per year, unless the foil is mechanicall
...
-The RBR sensor has a smaller form factor than the Aandera optodes, but is overall more similar to a 4831 with the temperature sensor very closely located to the sensing foil. 
+They also use further foil design (~1 s response) which they call “fast” . 
+The RBR sensor has a smaller form factor than the Aandera optodes, but is overall similar to a 4831 with the temperature sensor very closely located to the sensing foil. 
 This sensor has recently been implemented in gliders, and little is known about their performance.
 
 ### JFE Advantech RINKO
 AROD-FT sensor (RINKO JFE) is used for the SeaExplorer gliders (Alseamar) and for some Argo floats (small size and low power consumption) (see {numref}`ARODFT`). 
-This sensor is based on the optical (phosphorescence) principle which is now widely known as a remarkably fast response oxygen sensor (below 1s) with a high accuracy of 2 $\mu$mol/kg. 
+This sensor is based on the optical (phosphorescence) principle which a remarkably fast response oxygen sensor (below 1 s) with a good accuracy (±2 μmol kg<sup>-1</sup>). 
 This sensor used a multi-points calibration (16 points with 4 temperatures and 4 DO concentrations).
```
Comment (gkrahmann) [inline sections/oxygen_sensors_integration.md:88]: past tense ?   'used'  ?
Or should that be 'uses' ?
Also  'multi-points'  should be 'multi-point'

File: sections/oxygen_sensors_integration.md [hunk 4]
```diff
@@ -62,21 +72,21 @@ After this it should be less than 0.5 % per year, unless the foil is mechanicall
...
 This sensor has recently been implemented in gliders, and little is known about their performance.
 
 ### JFE Advantech RINKO
 AROD-FT sensor (RINKO JFE) is used for the SeaExplorer gliders (Alseamar) and for some Argo floats (small size and low power consumption) (see {numref}`ARODFT`). 
-This sensor is based on the optical (phosphorescence) principle which is now widely known as a remarkably fast response oxygen sensor (below 1s) with a high accuracy of 2 $\mu$mol/kg. 
+This sensor is based on the optical (phosphorescence) principle which a remarkably fast response oxygen sensor (below 1 s) with a good accuracy (±2 μmol kg<sup>-1</sup>). 
 This sensor used a multi-points calibration (16 points with 4 temperatures and 4 DO concentrations). 
-In this procedure, the DO reference standards are produced by saturing the primary mixtures with DO concentrations of approximately 4%, 10%, 17% and 25% respectively (certified by the National Metrology Institute of Japan). 
+The DO reference standards are produced by saturating the primary mixtures with DO concentrations of approximately 4%, 10%, 17% and 25% respectively (certified by the National Metrology Institute of Japan).
```
Comment (gkrahmann) [inline sections/oxygen_sensors_integration.md:89]: Are these reference standards for the multi-point calibration from the preceding sentence ? Then this should be
'The DO reference standards used for the multi-point calibrations are...'

**Review by gkrahmann — CHANGES_REQUESTED** [review]
I made some inline comments.

**Comment by soerenthomsen** [comment]
Thanks Gerd, @tomhull do you want to implement these and after we can merge I guess.

**Comment by gkrahmann** [comment]
all the rest looked fine

**Comment by soerenthomsen** [comment]
@tomhull do you agree with Gerds suggestions? If so I can also implement them and merge this PR

File: sections/oxygen_sensors_integration.md (hunk 2 above)
Comment (soerenthomsen) [reply sections/oxygen_sensors_integration.md:87]: @tomhull do you agree to replace `high` accuracy with `good` accuracy

Comment (gkrahmann) [reply sections/oxygen_sensors_integration.md:87]: there is now a mixup in the words.
replace   'which'   with 'creating'

**Comment by soerenthomsen** [comment]
Thx @tomhull. Please let me know when this can be merged!

**Pull request merged**